lib/v2/cnbeta/utils.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
                     content('.topic, .article-topic, .article-global').remove();
 
                     item.description = content('.article-summary').html() + content('.article-content').html();
+                    item.author = content('header.title div.meta span.source').text();
```

Comment:
Current feed will use the author field from https://www.cnbeta.com/backend.php which is 
 `.article-author` from the web page. This is more accurate than the source.

Reply:
I've checked the source, the author in rss.xml more refer to editor in cnbeta, but most of article in this source are reprint from other site.

Reply:
![image](https://user-images.githubusercontent.com/5434177/163860305-c76a6bc7-063a-45f3-ad5e-318b992d3ece.png)
![image](https://user-images.githubusercontent.com/5434177/163860364-bdbd5873-e0ff-45d6-ba27-f2ae6bda1000.png)
![image](https://user-images.githubusercontent.com/5434177/163860427-7a7e058f-91e8-4d9f-9e41-142cc64cc582.png)